packages/block-library/src/social-link/index.php

```diff
@@ -27,6 +27,14 @@ function render_block_core_social_link( $attributes, $content, $block ) {
 		return '';
 	}
 
+	/**
+	 * Prepend URL with https:// if it doesn't appear to contain a scheme
+	 * and it's not a relative link starting with //.
+	 */
+	if ( ! parse_url( $url, PHP_URL_SCHEME ) && ! str_starts_with( $url, '//' ) ) {
```

Comment:
Could this not be expressed as `if ( false === strpos( $url, '//' ) ) {` ? 🤔 
It would seem a bit simpler (and possibly even faster) that way...
If the string contains `//` we don't want to do anything because it either has the protocol defined, _or_ starts with `//`. So we basically only want to prepend `https://` if `//` is not contained in the string.

Reply:
Sure, but not all schemas contain `//`. So I would rather trust PHP on schema validity.